src/main/java/com/ioet/bpm/people/domain/UpdatePassword.java

```diff
@@ -0,0 +1,25 @@
+package com.ioet.bpm.people.domain;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@Data
+@DynamoDBTable(tableName = "update_password")
```

Comment:
We don't need any annotation related to the database, remove all the @Dynamo*** annotations in this class.